client/homebrew/phbStyle/phb.style.less

```diff
@@ -189,6 +189,7 @@ body {
 		border-image        : @noteBorderImage 11;
 		border-image-outset : 9px 0px;
 		box-shadow          : 1px 4px 14px #888;
+		transform           : translateZ(0);
```

Comment:
[Per this comment on the stackoverflow solution,](https://stackoverflow.com/questions/17881923/box-shadow-trimmed-in-css-columns-in-chrome#comment103838351_55110789) perhaps better to prefix with -webkit- so as to only kick in for Chrome users?

Reply:
We have a "Render Warning", that says that Homebrewery is built & tested for Chrome mostly, so I would expect the majority of users to use it and therefore, it doesn't seem like a critical fix to have; but yes, we can apply this change, I have no objections

Reply:
@G-Ambatte, do you want to make this change and then we'll merge it?

Reply:
I've pushed the change to `-webkit-transform` now.